src/main/java/core/basesyntax/dao/FruitDaoImpl.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.domain.Fruit;
+
+public class FruitDaoImpl implements FruitDao {
+    @Override
+    public void add(Fruit fruit) {
+        Storage.getFruits().add(fruit);
+    }
+}
```

Comment:
The `FruitDaoImpl` class should not directly interact with the Storage class to add a fruit. Instead, consider using a map or another collection within the DAO to manage the fruits. This will adhere to the encapsulation principle and make the DAO responsible for the persistence logic.